File: src/main/java/com/example/usercrud/business/config/RestTemplateConfig.java
package com.example.usercrud.business.config;

import com.example.usercrud.business.service.UserService;
import com.example.usercrud.business.service.UserServiceImpl;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.client.RestTemplate;

@Configuration
public class RestTemplateConfig {
    @Bean
    public RestTemplate restTemplate(){
        return new RestTemplate();
    }

}

File: src/main/kotlin/cloud/folium/usercrud/business/config/RestTemplateConfig.kt
package cloud.folium.usercrud.business.config

import org.springframework.web.client.RestTemplate
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class RestTemplateConfig {
    @Bean
    fun restTemplate(): RestTemplate {
        return RestTemplate()
    }
}